### PR TITLE
[Quantization]x86 pdll qgemm kernel wiout bias

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

File: tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc [hunk 1]
```diff
@@ -839,8 +929,10 @@ TAO_RAL_API("ral_pdll_qgemm", "cpu", ral_pdll_qgemm_acl_s8_s8_s8_per_channel);
 
 #if defined(TAO_X86)
 TAO_RAL_API("ral_qgemm", "cpu", ral_qgemm_onednn_s8_s8_s8_per_channel);
-TAO_RAL_API("ral_pdll_qgemm", "cpu",
-            ral_pdll_qgemm_onednn_s8_s8_s8_per_channel);
+TAO_RAL_API("ral_pdll_qgemm_s8s8s8f32_pc", "cpu",
+            ral_pdll_qgemm_acl_s8_s8_s8_f32_per_channel);
```
Comment (wyzero) [inline tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc:933]: Why using acl in `TAO_X86` section?

File: pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll [hunk 2]
```diff
@@ -70,7 +72,7 @@ Pattern TorchDequantLinearQuantOp {
     let infos = CreateTorchCustomCall(attr<"\"op\"">, inputs, outputs);
 
     /// 2. set attrs that are used by bladedisc.
-    SetAttr(infos.op, attr<"\"call_target_name\"">, attr<"\"ral_pdll_qgemm\"">);
+    SetAttr(infos.op, attr<"\"call_target_name\"">, attr<"\"ral_pdll_qgemm_s8s8s8f32_pc\"">);
     SetAttr(infos.op, attr<"\"input_placements\"">, attr<"\"h,h\"">);
```
Comment (wyzero) [inline pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll:76]: the number of items not equals to the number inputs.

File: pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll [hunk 3]
```diff
@@ -81,6 +83,83 @@ Pattern TorchDequantLinearQuantOp {
...
+  /// rewrite phase
+  rewrite output_quantize_op with {
+    /// 1. create custom call op
+    let inputs = PackValue_8(attr<"\"in\"">, input, weight, input_scale, input_zero_point, weight_scale, weight_zero_point, output_scale, output_zero_point);
+    let outputs = PackValue_1(attr<"\"out\"">, output_quantize_op.0);
+    let infos = CreateTorchCustomCall(attr<"\"op\"">, inputs, outputs);
+
+    /// 2. set attrs that are used by bladedisc.
+    SetAttr(infos.op, attr<"\"call_target_name\"">, attr<"\"ral_pdll_qgemm_s8s8s8_pc\"">);
+    SetAttr(infos.op, attr<"\"input_placements\"">, attr<"\"h,h\"">);
```
Comment (wyzero) [inline pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll:153]: ditto

File: tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc (hunk 1 above)
Comment (chenbohua3) [reply tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc]: have corrected:)

File: pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll (hunk 2 above)
Comment (chenbohua3) [reply pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll]: done

File: pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll (hunk 3 above)
Comment (chenbohua3) [reply pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll]: done

**Review by wyzero — DISMISSED** [review]

**Pull request merged**